docs/DeveloperGuide.adoc

```diff
@@ -721,17 +721,64 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |user |be able to sort my contacts both by their first name or last name |
 
-|`* * *` |user |add a new person |
+|`* * *` |user |have a "comment" section for each contact |record personal/miscellaneous information about the person
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |export all saved contacts and print out  |refer to when I can't use the software
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |add and create tags and add them to contacts |quickly find related personnels when needs be
 
-|`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
+|`* * *` |user |add and create tags and add them to contacts |quickly find related personnels when needs be
+
+|`* * *` |user |be able to edit the contact details |update when my contacts change their particulars
+
+|`* * *` |user |have the option to favourite some contacts |retrieve the information of contacts that I will frequently access
+
+|`* * *` |user |able to search my contacts by tags |know my relationship with the contact
+
+|`* * *` |user |add a person's details |use the saved details to contact him/her
+
+|`* * *` |new user |use the graphical interface |easily perform operations
+
+|`* * *` |new user |see tutorial bubbles the first time I use the software |familiarise myself with the features
+
+|`* * *` |forgetful user |use only a small part of a person's details to find his or her contacts|
```

Comment:
Just a suggestion from tutorial, maybe do not use "forgetful" to describe a user.  Maybe change to "a user does ... so that he does not need to remember ..."